addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -365,6 +365,9 @@ $truncate-limits: 2, 3, 10;
 }
 
 .o_wslides_course_main {
+    // Goal: push the footer out of the screen (in most use-case) to avoid seeing it moving when switching from course
+    // to review tab
+    min-height: 70vh;
```

Comment:
Should also be possible with bootstrap's `min-vh-70`, unless its used in multiple places?

Reply:
Yes, good idea to keep the inline styling with classes